api/model.go

```diff
@@ -189,6 +190,15 @@ type SupportBundleInitateInput struct {
 	Description string `json:"description"`
 }
 
+type BackupStatus struct {
+	client.Resource
+	Name        string `json:"id"`
+	Snapshot    string `json:"snapshot"`
+	Progress    int    `json:"progress"`
+	BackupURL   string `json:"backupURL"`
+	BackupError string `json:"backupError,omitempty"`
```

Comment:
Don't need `omitempty`. We don't have any other fields in the API with `omitempty`